misc/docerina/tools/hbs-designer/src/clients.hbs

```diff
@@ -25,7 +25,8 @@
     </div>
 
     <div class="fourteen wide column content-wrapper">
-      <h1>{{ clnt.name }}</h1>
+      <div class="main-wrapper">
+        <h1>{{ clnt.name }}</h1>
```

Comment:
Shall we use a meaningful name for variable ?

Reply:
This has been fixed